hack/coverage.sh

```diff
@@ -13,9 +14,32 @@ declare -a EXCLUDE_FILES_FROM_COVERAGE=("nodemaintenance_controller_init.go")
 # delete coverage files (if present)
 find . -name ${COVERAGE_FILE} | xargs rm -f
 
-# run ginkgo with coverage result line
-${GINKGO} ${GINKGO_ARGS} ./pkg/ ./cmd/ | sed  '/coverage:.*$/d'
-GSTAT=${PIPESTATUS[0]}
+REPEAT_COUNT=0
+MAX_REPEAT_COUNT=10
+while [[ $REPEAT_COUNT -lt $MAX_REPEAT_COUNT ]]; do
+
+		# run ginkgo with coverage result line
+		GINKGO_OUT=$(${GINKGO} ${GINKGO_ARGS} ./pkg/ ./cmd/ 2>&1 | sed  '/coverage:.*$/d')
```

Comment:
Are you sure that the `coverage:` lines do not contain any information about why this test is flaky?

Reply:
The lines taken out are from nodemaintenace_controller_init.go - these are functions that have zero coverage during unit testing. (like addition of controller to the controller runtime manager etc).
I checked that the problem happened without these changes at all, so i don't think there is a connection.

Reply:
i get it without this trick too; see https://github.com/onsi/ginkgo/issues/417

Reply:
I actually haven't seen these failures of the test runner for quite some time now.
It might have been some  problem on the side of the build server environment.